scripts/bootstrap-configure

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
 #
-#    Copyright 2014-2016 Nest Labs Inc. All Rights Reserved.
+#    Copyright 2014-2016,2020 Nest Labs Inc. All Rights Reserved.
```

Comment:
This should probably just be `Copyright 2020 Project nlbuild-autotools Authors. All Rights Reserved.` above this line.